TwitchChannelPointsMiner/TwitchChannelPointsMiner.py

```diff
@@ -28,6 +28,9 @@
     internet_connection_available,
     set_default_settings,
 )
+from colorama import init
+
+init()
```

Comment:
Check If this two line can be moved in `configure_loggers` (after method def)
https://github.com/Tkd-Alex/Twitch-Channel-Points-Miner-v2/blob/master/TwitchChannelPointsMiner/logger.py#L58

So we can `init()` only if `settings.colored is True`

Reply:
I've a little free time slot :)
I will do it

Reply:
Fixed it on the last commit!